### ISSUE TITLE
Bug Report: Spell Editor Crash + Inability to Display Chinese Skill Names

### ISSUE THREAD

**Issue by a480615** [pWn3d1337/Skyrim_SpellHotbar2#16]
Hello, I’m here to report multiple bugs.
1.An old save file that previously had SpellHotbar V1 installed causes the game to freeze and eventually crash to desktop (CTD) when opening the spell editor from the mod menu, even after clearing all SpellHotbar-related data using Resaver. However, this issue does not occur in old saves that never had SpellHotbar installed or in brand-new saves, where the spell editor opens without problems.
![QQ20241126-162432](https://github.com/user-attachments/assets/76b8f6c7-c3d0-4a60-94ab-607e99e02317)

2.SpellHotbar cannot display Chinese skill names. This issue has existed since V1, where all Chinese characters are replaced with a series of "????".
![c5a6c047-7f40-4a2e-ac5a-b9904042399a](https://github.com/user-attachments/assets/47c61925-ad32-40be-af0e-d1a60d2bbd3e)

**Comment by pWn3d1337** [comment]
1. Did you save the game without having a version of SpellHotbar installed?
2. known issue, support for other languages is difficult and not fully implemented.

**Comment by a480615** [comment]
> 1. Did you save the game without having a version of SpellHotbar installed?
> 2. known issue, support for other languages is difficult and not fully implemented.

Yes, I have saved the game after uninstalling SpellHotbar and cleaned the save file using Resaver, then saved the game again, but the issue still persists. Only save files that have never had any version of SpellHotbar installed, or entirely new save files, do not encounter this issue.

**Comment by pWn3d1337** [comment]
display of chinese character should now work with v0.0.5, please try again.

If spell editor still crashes, there should be a more accurate crash log to trace the issue.

**Comment by Hyacreap** [comment]
我试了一下，使用存档清理器后可以在旧存档使用。
我的环境是5x整合2.5+最新的烽火技能树+BFCO+环战技，所以应该不是这个mod的问题。
